lib/controller.js

```diff
@@ -285,6 +285,11 @@ module.exports = function createController(config) {
     });
   }
 
+  function shouldRefreshToken(token, serviceProviderConfig) {
```

Comment:
Waarom zit dit mee in de controller? Lijkt me eerder een service.

Reply:
Deze controle zat voordien ook al in de controller, ik heb nu een extra check toegevoegd op dezelfde locatie, enkel in een aparte functie gestoken.
De reden is, denk ik, dat dit afhankelijk is van de configuratie en die is beschikbaar in de controller maar niet in de service. Dan zou je de config moeten doorgeven aan de service.

Reply:
Ik ben het ook niet eens met Jb dat dit niet in de controller hoort. Tis de service die effectief de token gaat refreshen.
Als ge refreshToken callt op een service, verwacht ge effectief een nieuw access token, en niet dat hij niets doet.

Reply:
fair enough, dan laten we 't hier staan.